src/envoy/http/jwt_auth/pubkey_cache.h

```diff
@@ -83,6 +98,25 @@ class PubkeyCacheItem {
   }
 
  private:
+  // Searches "http://" or "https://" prefix in aud, and stores the position
+  // after protocol scheme prefix in beg. Searches trailing slash in aud, and
+  // stores the position before trailing slash in end. Returns true if aud has
+  // characters other than prefix and suffix.
+  bool SanitizeAudience(const std::string& aud, int& beg, int& end) {
```

Comment:
I think it is better for this function to return a processed string.
If not modification, (for most case), just return the input string which is not data copy (as I understand std::string) which is fast.

Reply:
Done.